include/hse/hse.h

```diff
@@ -57,7 +57,7 @@ extern "C" {
 
 #ifdef __has_attribute
 #  if __has_attribute(visibility)
-#    define HSE_EXPORT __attribute__((visibility("default")))
```

Comment:
While you are here, please fix the identation of the pre-processor commands

Reply:
Funny, it took everything I had not to do that.. 😄